templates/repo/user_cards.tmpl

```diff
@@ -19,6 +19,16 @@
 						<span class="octicon octicon-clock"></span> {{$.i18n.Tr "user.join_on"}} {{DateFmtShort .Created}}
 					{{end}}
 				</div>
+
+        {{if $.IsAdmin}}
+          <form class="display inline" action="{{$.RepoLink}}/action/unwatch?redirect_to={{$.Link}}" method="POST">
+ 				    {{$.CSRFTokenHTML}}
+ 				    <input type="hidden" name="user_id" value="{{.ID}}">
+						<div class="ui labeled button" tabindex="0">
+						  <button class="ui red small button">{{$.i18n.Tr "repo.unwatch"}}</button>
+						</div>
+					</form>
+   			{{end}}
```

Comment:
The indentation here seems mixed, I see spaces, tabs, and are not aligned properly.